src/markdown-pages/highlights.md

```diff
@@ -18,11 +18,31 @@ We have the following available Slack channels:
 Access the Galasa source code in [GitHub](https://github.com/galasa-dev) and open issues in the [project management repository](https://github.com/galasa-dev/projectmanagement).
 
 
-## 0.37.0 - Release Highlights
+## 0.38.0 - Release Highlights
+
+- CLI updates:
+
+  - You can get and delete secrets from the credentials store using the `galasactl secrets get` and `galasactl secrets delete` commands.
+
+    [Managing Ecosystem encryption keys](./docs/ecosystem/ecosystem-managing-encryption-keys.md) describes how to use `galasactl secrets get` to replace the encryption key being used to encrypt credentials in the Galasa Ecosystem's credentials store.
```

Comment:
secrets also work with the `galasactl apply -f <filename.yaml>` syntax

Reply:
Secrets also support base64 encoding so special characters are never going to be mangled by any codepage conversions...etc.